### DISCUSSION TITLE
clone playlist

### DISCUSSION THREAD

**Discussion by Marcioghiraldelli** [AzuraCast/AzuraCast#4698] [General]
I have radio "a" and radio "b" already in operation at all programming, and I want to copy only the playlists from radio "a" to radio "b" without having to delete radio "b" and use the clone button station. Is there any cli command in php or shell that I can copy only the playlist from one station to another?

**Comment by BusterNeece** [comment]
Currently, there's only a "clone playlist" function within a current station, and the ability to clone an entire station (including its playlists). Currently, you can't copy a playlist from one station to another directly, though it may be possible by exporting one station's playlist as a PLS/M3U file and then reimporting it from the other station.
